packages/dashboard/src/components/internalDashboard/index.tsx

```diff
@@ -279,6 +286,28 @@ const InternalDashboard: React.FC<InternalDashboardProperties> = ({ onSave, edit
     <TrendCursorSync>
       <TimeSync initialViewport={{ duration: '5m' }} group='dashboard-timesync'>
         {readOnly ? ReadOnlyComponent : EditComponent}
+        <ConfirmModal
```

Comment:
Can we refactor this to its own component? Something like `<ConfirmDeleteModal />`. Trying to keep the complexity of Internal dashboard to a minimum

Reply:
renamed to ConfirmDeleteModal